OBSDOCS-219 - Formatting fix - draft (#67696)
openshift/openshift-docs: modules/logging-loki-gui-install.adoc

.Procedure

> . To install the `Loki Operator` in the {product-title} web console, click *Operators* -> *OperatorHub*.

Reviewer: ```suggestion
. To install the {loki-op} in the {product-title} web console, click *Operators* -> *OperatorHub*.
```
We have an attribute for this now 🙏🏻

. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.

> [NOTE]

Reviewer: The `+` needs to be here I think, and the same for the following steps where it was removed. See https://67696--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-loki#logging-loki-gui-install_cluster-logging-loki

====
The Community Loki Operator is not supported by Red Hat.
====

> . On the Install Operator page, for *Update Channel* select *stable*.

Reviewer: Pretty sure this is "**Update channel**" now, please check the capitalization in the console. I think I had a Jira opened recently to fix this in a different module.
```suggestion
. On the *Install Operator* page, for *Update channel* select *stable*.
```

+
--
include::snippets/logging-stable-updates-snip.adoc[]
--

As the Loki Operator must be deployed to the global operator group namespace `openshift-operators-redhat`, *Installation mode* and *Installed Namespace* is already be selected. If this namespace does not already exist, it is created for you.

. Select *Enable operator-recommended cluster monitoring on this namespace.*

This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.

. For *Update approval* select *Automatic*, then click *Install*.

--
include::snippets/logging-approval-strategy-snip.adoc[]
--

> . To install the `Red Hat OpenShift Logging` Operator in the {product-title} web console, click *Operators* -> *OperatorHub*.

Reviewer: ```suggestion
. To install the {clo} in the {product-title} web console, click *Operators* -> *OperatorHub*.
```
We have an attribute for this now 🙏🏻

.Procedure

. To install the {loki-op} in the {product-title} web console, click *Operators* -> *OperatorHub*.

. Type {loki-op} in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
+
[NOTE]
====
The Community Loki Operator is not supported by Red Hat.
====

. On the *Install Operator* page, for *Update channel* select *stable*.
+
[NOTE]
====
The `stable` channel only provides updates to the most recent release of logging. To continue receiving updates for prior releases, you must change your subscription channel to `stable-X` where `X` is the version of logging you have installed.
====
+

> As the {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, *Installation mode* and *Installed Namespace* is already be selected. If this namespace does not already exist, it is created for you.

Reviewer 2: ```suggestion
As the {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, *Installation mode* and *Installed Namespace* is already selected. If this namespace does not already exist, it is created for you.
```

The Operators should now be available to all users and projects that use this cluster.

. Verify the operator installations:
. Navigate to *Operators* -> *Installed Operators*.
. Make sure the *openshift-logging* project is selected.

> . In the *Status* column, verify that you see green checks with *InstallSucceeded* and the text *Up to date*, below.

Reviewer: Please revise this sentence; we don't use "below" generally for accessibility per the IBMSG:
![image](https://github.com/openshift/openshift-docs/assets/40172997/eef8fb8d-a1b5-4c1a-a7f2-cb5ef5b14ffc)
```suggestion
. In the *Status* column, verify that you see green checkmarks with *InstallSucceeded* and the text *Up to date*.
```
